### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by JBBianchi**

### CODE REVIEW

**Comment by JBBianchi** [comment]
@ricardozanini @cdavernas I think some config is missing in this repo as you weren't assigned as reviewer by default.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- **[Synapse](https://github.com/serverlessworkflow/synapse)**: Our official, open source, vendor-neutral runtime implementation, facilitating seamless deployment and operation of workflows.
+
+### Join Our Community
+
+We thrive on community contributions and are always eager to welcome new members. Whether you're interested in developing new SDKs, enhancing existing tools, or sharing your insights, there's a place for you here.
+
+Connect with us on:
+- [GitHub Discussions](https://github.com/serverlessworkflow/specification/discussions)
+- [Slack](https://slack.cncf.io/)
+- [X/Twitter](https://twitter.com/ServerlessWF)
```
Comment (ricardozanini) [inline README.md]:
```suggestion
- [X/Twitter](https://twitter.com/CNCFWorkflow)
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- **[SDKs](https://github.com/serverlessworkflow/specification#sdks)**: Access our SDKs for [Go](https://github.com/serverlessworkflow/sdk-go), [Java](https://github.com/serverlessworkflow/sdk-java), [.NET](https://github.com/serverlessworkflow/sdk-net), [Python](https://github.com/serverlessworkflow/sdk-python), and [TypeScript](https://github.com/serverlessworkflow/sdk-typescript) to interact with workflows in your preferred programming language.
+- **[Synapse](https://github.com/serverlessworkflow/synapse)**: Our official, open source, vendor-neutral runtime implementation, facilitating seamless deployment and operation of workflows.
+
+### Join Our Community
+
+We thrive on community contributions and are always eager to welcome new members. Whether you're interested in developing new SDKs, enhancing existing tools, or sharing your insights, there's a place for you here.
+
+Connect with us on:
+- [GitHub Discussions](https://github.com/serverlessworkflow/specification/discussions)
+- [Slack](https://slack.cncf.io/)
```
Comment (ricardozanini) [inline README.md]:
```suggestion
- [Slack](https://slack.cncf.io/) - Channel #serverless-workflow
```

File: README.md (hunk 1 above)
Comment (JBBianchi) [reply README.md]: Do we still have access to `CNCFWorkflow` ?

Comment (ricardozanini) [reply README.md]: Oh yeah, I've recovered the username.

**Review by cdavernas — APPROVED** [review]
Looks great to me! Thanks man ❤️

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**